### ISSUE TITLE
Confirm: vmime does not currently support parsing lists in the from header

### ISSUE THREAD

**Issue by bmagistro** [kisli/vmime#318]
From what I can tell looking at the code, vmime will parse a single address in the from header while the spec allows for the from header to contain a list of addresses.  At this time just looking to confirm this is the correct understanding/current behavior.

* https://datatracker.ietf.org/doc/html/rfc5322#section-3.6.2
* https://serverfault.com/questions/554520/smtp-allows-for-multiple-from-addresses-in-the-rfc-was-this-ever-useful-why-do

**Comment by vincent-richard** [comment]
That's right, "From" field only supports a single mailbox, as in the original RFC-822 (it is a mailbox list since RFC-2822). It could be changed easily in "headerFieldFactory" but problem is that this is an API-breaking change.

**Comment by bmagistro** [comment]
The way I read 822 it supported lists too.  Agree the change would be a break to the existing api.  based on other comments I'm not sure if it is something we will need.  If it does turn out to be a thing, would you be open to a PR for that or should we plan to maintain a local patch?

from rfc 822 https://datatracker.ietf.org/doc/html/rfc822#section-4.1
```
     authentic   =   "From"       ":"   mailbox  ; Single author
                 / ( "Sender"     ":"   mailbox  ; Actual submittor
                     "From"       ":" 1#mailbox) ; Multiple authors
                                                 ;  or not sender
```

**Comment by vincent-richard** [comment]
Hi! Sorry for not replying sooner, last weeks were quite busy.

Actually, we could take advantage of the fact that API is not considered stable yet (as version 1.0 has not been released), and I think a PR would be useful as I have always strived to ensure that VMime complies with standards.
